src/services/OfferService.js

```diff
@@ -1,58 +1,55 @@
 import Offer from '../models/Offer'
-import fs from 'fs'
 import hash from '../utils/hash'
 import createToken from '../utils/createToken'
 import MailService from './MailService'
+import forms from '../models/forms'
 
 const MILLISECONDS_IN_A_DAY = 1000 * 60 * 60 * 24
 
 export default class OfferService {
-  constructor (fileName) {
-    this.fileName = fileName
-    this.offers = this.read()
-  }
-
-  createNewId () {
-    if (this.offers.length === 0) {
-      return 0
-    } else {
-      return this.offers[this.offers.length - 1].id + 1
-    }
-  }
-
   async createOffer (city, email, formData, duration) {
-    const id = this.createNewId()
     const token = createToken()
+
+    const form = new forms[city](formData)
     const offer = new Offer({
-      id,
-      email,
-      city,
-      formData,
+      email: email,
+      city: city,
       expirationDate: Date.now() + duration * MILLISECONDS_IN_A_DAY,
```

Comment:
Does it make sense to use moment in the backend? If yes create a task for this.

Reply:
Rather not, we probably would have to care more about converting from and to the moment objects 🤔